plugins/dataTypes/Boolean/Boolean.class.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+/**
+ * @package DataTypes
+ */
+
+class DataType_Boolean extends DataTypePlugin {
+
+	protected $isEnabled = true;
+	protected $dataTypeName = "Boolean";
+	protected $hasHelpDialog = true;
```

Comment:
This @hasHelpDialog line can be removed. Its legacy code from earlier versions of the script.